elasticdl/pkg/ps/server.go

```diff
@@ -2,52 +2,110 @@ package ps
 
 import (
 	"context"
+	"elasticdl.org/elasticdl/pkg/common"
 	pb "elasticdl.org/elasticdl/pkg/proto"
+	"fmt"
 	empty "github.com/golang/protobuf/ptypes/empty"
 	"google.golang.org/grpc"
 	"log"
 	"net"
+	"sync"
 )
 
-type psServer struct {
+// Server defines servicer of ps
+type Server struct {
 	pb.PserverServer
+	Param *Parameter
+	Opt   Optimizer
+	PSID  int32
```

Comment:
PSID => ID, because the fullname is `ps.Server.ID`

Reply:
Done